src/search/containers/no-results/no-results.component.spec.ts

```diff
@@ -1,16 +1,41 @@
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { ActivatedRoute, convertToParamMap, ParamMap, Params } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { Observable, ReplaySubject } from 'rxjs';
+import { NoResultsMessageId } from 'src/search/enums';
 import { NoResultsComponent } from './no-results.component';
 
+export class ActivatedRouteStub {
+  // Use a ReplaySubject to share previous values with subscribers
+  // and pump new values into the `paramMap` observable
+  private subject = new ReplaySubject<ParamMap>();
```

Comment:
@Daniel-Lam - We no longer need this class, as I have simplified the code to use ActivatedRoute snapshot instead of instead of subscribing to the ActivatedRoute paramMap.